pom.xml

```diff
@@ -27,14 +34,63 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <docker.image.prefix>fwsbac</docker.image.prefix>
         <spring.cloud.config.version>1.2.2.RELEASE</spring.cloud.config.version>
+        <mockito.version>1.10.19</mockito.version>
+        <joda-time.version>2.9.5</joda-time.version>
 
-        <tds.config.client.version>0.0.1-SNAPSHOT</tds.config.client.version>
         <tds.common.version>0.0.1</tds.common.version>
         <tds-common-legacy.version>0.0.1</tds-common-legacy.version>
-        <tds.student.client.version>0.0.1-SNAPSHOT</tds.student.client.version>
```

Comment:
Yessir, and it's a good thing, too.  I didn't see any circular dependencies in my once-over.